index.html

```diff
@@ -7,57 +7,215 @@
+                            through some of my work and experience! If you like what 
+                            you see and have a project you need coded,
+                            don't hesitate to contact me.
+                        </p>
+                        <!-- social media section -->
+                            <ul class="image-icons">
+                                <li> <a href="https://github.com/mailsg" target="_blank" rel="noopener noreferrer">
+                                    <img src="images/GitHub.png" alt="GitHub"></a> 
+                                </li>
+                                <li> <a href="https://www.linkedin.com/in/sandeep-ghosh-660830166/" target="_blank" rel="noopener noreferrer">
+                                    <img src="images/Linkedin.png" alt="Linkedin"></a> 
+                                </li>
+                                <li> <a href="https://twitter.com/microgeniusme" target="_blank" rel="noopener noreferrer">
+                                    <img src="images/Twitter.png" alt="Twitter"></a> 
+                                </li>
+                                <li> <a href="https://angel.co/" target="_blank" rel="noopener noreferrer">
+                                    <img src="images/Angelist.png" alt="Angelist"></a> 
+                                </li>
+                                <li> <a href="https://medium.com/" target="_blank" rel="noopener noreferrer">
+                                    <img src="images/Medium.png" alt="Medium"></a> 
+                                </li>
+                            </ul>
 
+                    </div>
+                
+            </section>
+            <!-- works section -->
+            <section class="works">
+                    <article class="grid-container">
+                        <h2>My Recent Works</h2>
+                        <div class="divider1">
+                            <img src="images/vector.png" alt="vector">
+                        </div>
+                        <div class="image"></div>
+                        <div class="card-body">
+                            <h5>Multi-Post Stories</h5>
+                                <p>A daily selection of privately
+                                    personalized reads; no accounts or sign-ups required.
+                                    has been the industry's standard dummy text ever since
+                                    the 1500s, when an unknown printer took a standard dummy text.
+                            </p>
+                            <ul>
+                                <li class="css">CSS</li>
+                                <li class="html">HTML</li>
+                                <li class="bootstrap">Bootstrap</li>
+                                <li class="ruby">Ruby</li>
+                            </ul>
+                            <button type="button">See project</button>
```

Comment:
- [ ] Kindly ensure that your buttons also follow the design guidelines, such as `enabled, `hover`, `pressed` and `disabled`, so as to have a pixel-perfect design like you'll soon be doing in your job 🙂.

- Your design
![mobile](https://user-images.githubusercontent.com/41163571/227705513-e7b75a73-8c6a-4e3d-9514-eb2f49bdfae2.gif)

- Figma design
<img width="274" alt="button2" src="https://user-images.githubusercontent.com/41163571/227492511-ac99583d-add1-41a8-b99f-2b2f12771941.png">


- For the three types of buttons I went through.